benchmarks/src/test/java/org/apache/druid/benchmark/query/SqlNestedDataBenchmark.java

```diff
@@ -199,7 +199,7 @@ public String getFormatString()
       // 42, 43 big cardinality like predicate filter
       "SELECT SUM(long1) FROM foo WHERE string5 LIKE '%1%'",
       "SELECT SUM(JSON_VALUE(nested, '$.long1' RETURNING BIGINT)) FROM foo WHERE JSON_VALUE(nested, '$.nesteder.string5') LIKE '%1%'",
-      // 44, 45 big cardinality like filter + selector filter
+      // 44, 45 big cardinality like filter + selector filter with different ordering
```

Comment:
Ran the tests for 44,45,46,47. The scores are very similar, since we've the ordering:

```
Benchmark                        (query)  (rowsPerSegment)  (schema)  (stringEncoding)  (vectorize)  Mode  Cnt  Score   Error  Units
SqlNestedDataBenchmark.querySql       44           5000000  explicit              none        force  avgt    5  7.753 ± 0.380  ms/op
SqlNestedDataBenchmark.querySql       44           5000000      auto              none        force  avgt    5  8.023 ± 0.940  ms/op
SqlNestedDataBenchmark.querySql       45           5000000  explicit              none        force  avgt    5  7.976 ± 0.735  ms/op
SqlNestedDataBenchmark.querySql       45           5000000      auto              none        force  avgt    5  7.820 ± 0.863  ms/op
SqlNestedDataBenchmark.querySql       46           5000000  explicit              none        force  avgt    5  7.495 ± 0.279  ms/op
SqlNestedDataBenchmark.querySql       46           5000000      auto              none        force  avgt    5  7.861 ± 0.691  ms/op
SqlNestedDataBenchmark.querySql       47           5000000  explicit              none        force  avgt    5  7.577 ± 0.405  ms/op
SqlNestedDataBenchmark.querySql       47           5000000      auto              none        force  avgt    5  7.735 ± 0.491  ms/op
```

Reply:
Comparing with upstream/master branch:
```
Benchmark                        (query)  (rowsPerSegment)  (schema)  (stringEncoding)  (vectorize)  Mode  Cnt    Score   Error  Units
SqlNestedDataBenchmark.querySql       44           5000000  explicit              none        force  avgt    5  215.894 ± 2.712  ms/op
SqlNestedDataBenchmark.querySql       44           5000000      auto              none        force  avgt    5  208.718 ± 5.545  ms/op
SqlNestedDataBenchmark.querySql       45           5000000  explicit              none        force  avgt    5  221.829 ± 7.157  ms/op
SqlNestedDataBenchmark.querySql       45           5000000      auto              none        force  avgt    5  216.632 ± 2.712  ms/op
SqlNestedDataBenchmark.querySql       46           5000000  explicit              none        force  avgt    5    7.431 ± 0.286  ms/op
SqlNestedDataBenchmark.querySql       46           5000000      auto              none        force  avgt    5    7.396 ± 0.186  ms/op
SqlNestedDataBenchmark.querySql       47           5000000  explicit              none        force  avgt    5    7.487 ± 0.287  ms/op
SqlNestedDataBenchmark.querySql       47           5000000      auto              none        force  avgt    5    7.451 ± 0.203  ms/op

```